cmd/create/runner.go

```diff
@@ -162,7 +164,7 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 	var release v1alpha1.Release
 	var provider string
 	var releaseVersion string
-	r.logger.LogCtx(context.Background(), "message", "determining release to test")
```

Comment:
Took the opportunity to use the actual context instead of `context.Background()`.